Resources/Prototypes/Entities/Objects/Misc/kudzu.yml

```diff
@@ -0,0 +1,56 @@
+- type: entity
+  id: Kudzu
+  name: Kudzu
+  description: A rapidly growing, dangerous plant. WHY ARE YOU STOPPING TO LOOK AT IT?!
+  placement:
+    mode: SnapgridCenter
+    snap:
+      - Wall
+  components:
```

Comment:
you should give Kudzu ReactiveComponent with the Flammable group so you can hellfoam it to death